app/build.gradle

```diff
@@ -54,10 +54,10 @@ android {
 }
 
 dependencies {
-    implementation 'androidx.appcompat:appcompat:1.5.1'
+    implementation 'androidx.appcompat:appcompat:1.6.0'
```

Comment:
Correct me if I am wrong.
We are just updating the gradle files, not adding BOM. 
Because this is views branch

Reply:
That's correct, I misnamed this. This is more for keeping dependencies consistent.